src/massive/munit/TestRunner.hx

```diff
@@ -182,29 +195,88 @@ class TestRunner implements IAsyncDelegateObserver
         suiteIndex = 0;
         clientCompleteCount = 0;
         Assert.assertionCount = 0; // don't really like this static but can't see way around it atm. ms 17/12/10
-        testSuites = [for(suiteType in testSuiteClasses) Type.createInstance(suiteType, emptyParams)];
+        emptyParams = new Array();
```

Comment:
Why duplicate the initialization of this variable?

Reply:
Yep... this looks redundant compared with the current source